src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,129 @@
+package duke;
+
+import java.util.List;
+import java.util.Scanner;
+
+import duke.tasks.Task;
+
+
+/**
+ * UI class that is responsible for reading user input and printing
+ */
+public class Ui {
+    /**
+     * Constructor to initialize the UI Class
+     */
+    public Ui() {
+    }
+    
+    /**
+     * Prints the welcome message.
+     */
+    public static String printWelcome() {

```

Comment:
The methods here should be renamed to reflect the actual use of the function. No printing is done within the method. e.g getWelcomeMessage(), getByeMessage()

Reply:
Noted, will change to avoid confusion